docs/kintoblocks.md

```diff
@@ -15,15 +15,26 @@ We take off your shoulders the hassle of library configuration, deployment, and
 Creating your first KintoBlock is super easy.
 
 1. Select **KintoBlocks** from the sidebar
+
 2. Click **Create New KintoBlock**
 ![Screenshot - Create New KintoBlock](/docs/assets/creating-a-kintoblock-1-2.png)
-3. Enter the required info
-4. Click **Create New KintoBlock**
+
+3. Select the type of KintoBlock : We have three types of KintoBlock in the pipeline but currently you can select either **Microservice** or **Website**
+
+4. Enter the required info. This form will differ depending on the **type** of KintoBlock.
+
+5. Click **Create New KintoBlock**
 ![Screenshot - Enter Info](/docs/assets/creating-a-kintoblock-3-4.png)
 _That's it! You have created your first KintoBlock!_
+
 6. On GitHub, go to the repository associated with this KintoBlock
 ![Screenshot - New GitHub Repo](/docs/assets/creating-a-kintoblock-5.png)
 
+### Points to remember about Creating a Website
+
+- You can use static files, with an `index.html` file for entry.
```

Comment:
Can kintohub docs be used an an example docker file and deployd with kintohub? Otherwise we should actually show an example docker file here and not just reference NPM.

Reply:
NPM? the docker example is from our static site, but I do not understand exactly what you mean.

Reply:
any custom dockerfile can be used, need to make sure file builds in 'build'

remove lines after build (99)